pypika/terms.py

```diff
@@ -335,25 +337,27 @@ def __init__(self, value: Any, alias: Optional[str] = None) -> None:
         super().__init__(alias)
         self.value = value
 
```

Comment:
I see what you are trying to do with the UNSET object to get around the valid None value, however I think extracting the contents of the `get_value_sql` into a different method would be a more simple approach.

For example, you could have a static `get_formatted_value` (or another name) method which implements the if branches that format and return the value. Then `get_value_sql` would just need to call that with the correct value: for Enum value.value or just self.value otherwise.

That would prevent the need to use the UNSET hack. Does this make sense?

Reply:
I rather dislike static methods, but as I understand a classmethod would do just as well and even works with inheritance and stuff?

Reply:
PR is updated.

btw, how do you prefer PR updates. Can I use force-pushs directly or use fixup commits and rebase in the end?

Reply:
Either way. I generally like to merge in single commit so the history remains clean.

Understood regarding classmethod in this use case. Now I look at it again it makes more sense than the staticmethod approach I originally suggested given it avoids the need to handle enums differently in `get_value_sql`!